### PR TITLE
Components

### PR DESCRIPTION
**Pull request by gemmen29**
In this PR I added the following:

1. Use class component to create calculator component.
2. Add project styles.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
```
Comment (TedLivist) [inline src/App.js]: - [x] As stated in the instructions, your components should be class-based. Kindly change this functional **`App.js`** component to a class-based one.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gemmen29,

Good job so far!
✔️ Nice documentation
✔️ Clean code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] According to the instructions, the boilerplate files from CRA should have been deleted. Kindly delete the files as presented in the image below:

  magdy-cra
- [x] You seem to have made a mistake in your README instructions (see image below). In item 5, I think you meant **`npm i`**. Also, add another instruction on how to run the application after installing the packages ie. **`npm start`**

  magdy-readme

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (gemmen29) [reply src/App.js]: Done.

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @gemmen29 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**